config/development.toml

```diff
@@ -202,9 +202,15 @@ disabled = false
 consumer_group = "SCHEDULER_GROUP"
 
 [email]
-from_email = "notify@example.com"
+sender_email = "example@example.com"
 aws_region = ""
-base_url = ""
+base_url = "http://localhost:8080"
+allowed_unverified_days = 1
+active_email_client = "SES"
+
+[email.aws_ses_config]
```

Comment:
Then this would also be `email.aws_ses`.
And the corresponding environment variable would be `ROUTER__EMAIL__AWS_SES__EMAIL_ROLE_ARN` etc.